docs/new-architecture-library-intro.md

```diff
@@ -469,6 +484,23 @@ const RCTWebViewNativeComponent: HostComponent<mixed> =
   requireNativeComponent < mixed > 'RNTMyNativeView';
 ```
 
+#### Later on you can replace `requireNativeComponent`
+
+When you are ready to migrate to Fabric you can replace `requireNativeComponent` with `codegenNativeComponent`:
+
+```ts title="RNTMyNativeViewNativeComponent.js"
+export default (codegenNativeComponent<NativeProps>(
+   'RNTMyNativeView',
+): HostComponent<NativeProps>);
+```
+
+And update the main file:
+
+```ts title="RNTMyNativeNativeComponent.js"
+export default require('./RNTMyNativeViewNativeComponent')
+  .default;
```

Comment:
I think would be better not to mix commonjs & ES module

Reply:
so, the proper way will be
```
module.exports = require('./RNTMyNativeViewNativeComponent').default
```
?

Reply:
@cipolleschi maybe

```js
export { default } from './RNTMyNativeViewNativeComponent'
```